cli_tools/gce_vm_image_import/importer/importer_test.go

```diff
@@ -102,130 +73,18 @@ func TestGetWorkflowPathsFromFile(t *testing.T) {
 	workflow, translate := getWorkflowPaths(source, dataDisk, osID, customTranWorkflow, currentExecutablePath)
 
 	if workflow != homeDir+WorkflowDir+ImportAndTranslateWorkflow {
-		t.Errorf("resulting workflow path `%v` does not match expected `%v`", workflow, homeDir+WorkflowDir+ImportAndTranslateWorkflow)
+		t.Errorf("resulting workflow Path `%v` does not match expected `%v`", workflow, homeDir+WorkflowDir+ImportAndTranslateWorkflow)
```

Comment:
Why is Path capitalized?

Reply:
I think it's renamed by-the-way when renaming function path() to Path(). Eric, there may be many this kind of accidental renaming in text/comment.

Reply:
Oh gross, yeah, you're right. Thanks for catching that -- I'll fix the other instances.

Reply:
done